python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -170,7 +170,10 @@ def get_assets_live_info(graphene_info, step_keys_by_asset: Mapping[AssetKey, Li
             list(unstarted_run_ids_by_asset.get(asset_key, [])),
             list(in_progress_run_ids_by_asset.get(asset_key, [])),
             GrapheneRun(run_records_by_run_id[latest_run_ids_by_asset[asset_key]])
+            # Dagit error occurs if a run is terminated at the same time that this endpoint is called
```

Comment:
this doesn't quite make sense to me... 

run_records_by_run_id is calculated by fetching all the run_records based on the provided run_ids.  Why does run termination affect anything?

Reply:
Ah, instead of terminated I meant deleted. The user stated that the run disappeared from the runs page, which implies the run was deleted.